<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AWS IAM : Python Qt</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>7</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Amazon Web Services</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IAM Services</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>421</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Access Key Id</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="access_key_id_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Secret Access Key</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="secret_access_key_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="region_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>323</x>
      <y>47</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User : </string>
    </property>
   </widget>
   <widget class="QLabel" name="access_user">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>48</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
